.github/workflows/keep-an-eye-on-ruby-head.yml

```diff
@@ -14,6 +14,7 @@ jobs:
           - "windows-latest"
         ruby-version:
           - "head"
+          - "truffleruby-head"
```

Comment:
I think we'll need an exclude here too for windows + truffleruby-head

Reply:
Oh, I had forgotten that the project [has already a daily workflow](https://github.com/fxn/zeitwerk/blob/main/.github/workflows/keep-an-eye-on-truffleruby-head.yml) for `truffleruby-head`! I have removed that line.